content/1-introduction-3-problems.md

```diff
@@ -0,0 +1,129 @@
+## Open problems {#sec:intro-problems}
```

Comment:
### Camille
Remarque mineure, la mise en scène des questions ouvertes comme "If the two approaches aim to explain different features of culture, in which domain is each approach most appropriate? Are there domains that involve both imitation and non-copying communication in important degrees? How do the two processes interact in such cases? These are the questions left open by recognising a complementarity between the Parisian and Californian approaches." est peut-être un peu chiche, might be useful to make it a bit more detailed, de manière à comprendre dans quel contexte ça pourrait valoir le coup de répondre à ces questions.  Si c'est aussi le cœur même de la question (de trouver de tels exemples concrets), on peut laisser de côté.